# HomingSequence.srv

# Request (empty request in this case)
---

# Response
float32 x_limit
float32 y_limit